libs/ui/src/modals/screens/add-account/components/private-key/private-key.styles.ts

```diff
@@ -22,11 +22,14 @@ export const styles = StyleSheet.create({
   inputContainer: {
     width: '100%'
   },
+  inputInnerContainer: {
+    height: getCustomSize(14.75)
+  },
   textarea: {
-    height: getCustomSize(14.75),
-    marginTop: getCustomSize(1.25),
-    marginBottom: getCustomSize(3),
-    paddingRight: getCustomSize()
+    height: '100%',
+    paddingRight: getCustomSize(4.25),
+    paddingTop: getCustomSize(1.75),
+    paddingBottom: getCustomSize(1.75)
```

Comment:
```paddintVertical``` doesn't correctly work on the ios simulator, and I changed it to separate styles